src/ssl.c

```diff
@@ -15537,6 +15646,101 @@ char* wolfSSL_ASN1_TIME_to_string(WOLFSSL_ASN1_TIME* t, char* buf, int len)
 }
 #endif /* WOLFSSL_MYSQL_COMPATIBLE */
 
+#if defined(OPENSSL_EXTRA) && !defined(NO_ASN_TIME) \
+&& !defined(USER_TIME) && !defined(TIME_OVERRIDES)
+
+  #if !defined(XSNPRINTF)
+      #define XSNPRINTF snprintf
```

Comment:
XSNPRINTF is mapped to snprintf in ./wolfssl/wolfcrypt/types.h (to either snprintf, or _snprintf on Windows).  Why does it need to be remapped here?